### PR TITLE
Character Creation 

### PR DESCRIPTION
**Pull request by Kravos**

### CODE REVIEW

File: Program.cs [hunk 1]
```diff
@@ -1,12 +1,85 @@
...
             Console.WriteLine("Welcome to my RPG");
+            
+            
+            Console.WriteLine("---- Menue ----");
+            Console.WriteLine("Enter one of the following options\n");
+            Console.WriteLine("'create' - to create a new character.");
+            Console.WriteLine("'exit' - to leave.\n");
+
+            var menueInput = Console.ReadLine();
+            if (menueInput.Equals("exit", StringComparison.OrdinalIgnoreCase))
```
Comment (BlowaXD) [inline Program.cs]: switch case

File: Program.cs [hunk 2]
```diff
@@ -1,12 +1,85 @@
...
+            //Nickname
+            Console.WriteLine("Please enter your Nickname:");
+            var nicknameInput = Console.ReadLine();
+            player.Nickname = nicknameInput;
+
+            //Class
+            Console.WriteLine("\nPlease enter your class! Choose between: Human, Monster and Mystic");
+
+            player.Race = Console.ReadLine();
+            String[] classes = {"Human" , "Monster" , "Mystic"};
```
Comment (BlowaXD) [inline Program.cs]: dictionary or hashset

Comment (BlowaXD) [inline Program.cs]: Use enum as ClassType

File: Program.cs (hunk 1 above)
Comment (BlowaXD) [reply Program.cs]: Use enum as CommandType

**Comment by Kravos** [comment]
Changes applied

**Pull request merged**